service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorServiceImpl.java

```diff
@@ -176,9 +180,23 @@ private void validateAdapterEntityOrFail(final AttributeAdapterConnection adapte
         if (adapter.getAdapterEndpoint() == null || adapter.getAdapterEndpoint().isEmpty()) {
             throw new AttributeConnectorException("Attribute adapter configuration requires a nonempty endpoint URL");
         }
+        try {
+            if (!new URI(adapter.getAdapterEndpoint()).getScheme().equalsIgnoreCase(HTTPS)) {
```

Comment:
Good point regarding missing schemes.. converted to using `java.net.URL` so that if a protocol isn't specified it will throw a `MalformedURLException` (which also means no defensive checks necessary)